devel/bazel/Portfile

```diff
@@ -49,8 +49,10 @@ distname                  ${distname}-dist
 dist_subdir               ${name}
 
 # Checksum Phase
-checksums                 rmd160  5269d2fce917d3324dcb75ca06fe12089a4e24a1 \
-                          sha256  2418c619bdd44257a170b85b9d2ecb75def29e751b725e27186468ada2e009ea
+checksums                 rmd160  ed9c36b49d60aaead2d16bc28ece7c026745d632 \
+                          sha256  dfd0761e0b7e36c1d74c928ad986500c905be5ebcfbc29914d574af1db7218cf
+depends_build-append \
```

Comment:
Minor nitpick, can you add an empty line before `depends_build-append`?

Reply:
Yes, done and squashed commits.